docs/releases/1.10.1.txt

```diff
@@ -29,3 +29,6 @@ Bugfixes
 
 * Fixed the ``isnull`` lookup on a ``ForeignKey`` with its ``to_field``
   pointing to a ``CharField`` (:ticket:`26983`).
+
+* Fixed updating ``obj._state`` when doing ``bulk_create`` with a backend that
```

Comment:
Made bulk_create() properly initialize model instances on backends such as PostgreSQL that support ...